syne_tune/optimizer/schedulers/searchers/gp_fifo_searcher.py

```diff
@@ -380,7 +380,7 @@ def _assign_random_searcher(self, random_seed=None):
             if random_seed is None:
                 random_seed = self.random_state.randint(0, 2 ** 32)
             self._random_searcher = RandomSearcher(
-                self.hp_ranges.config_space,
+                self.hp_ranges.config_space_for_sampling,
```

Comment:
Could you also import the unit test I added in the other PR - to make sure that this will work as expected going forward?